cs-offerings/kube-configs/blockchain-couchdb.yaml

```diff
@@ -350,7 +354,7 @@ spec:
     spec:
       containers:
       - name: couchdb1
-        image: ibmblockchain/fabric-couchdb:0.4.6
+        image: ibmblockchain/fabric-couchdb:0.4.10
```

Comment:
I believe this is the wrong version of fabric-couch for fabric 1.1. 0.4.6 is for fabric 1.1, 0.4.10 is for fabric 1.2

Reply:
As per https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh which is referenced in https://hyperledger-fabric.readthedocs.io/en/release-1.1/samples.html, third party images are tagged at 0.4.10

Reply:
The bootstrap script is in master and is for all versions of fabric. The default for that script is for the current release of 1.2 which pre-reqs 0.4.10. For 1.1 the pre-req version is 0.4.6.
see https://github.com/hyperledger/fabric/blob/095f8815a8ea1e3450595a76e0a464416a145692/Makefile#L47
which is the fabric 1.1 makefile for example.

Reply:
Updated the version based on the above reviewer comment. Please pull the latest version for review